Adding a section dealing with ransomware protection (#13163)
MicrosoftDocs/azure-devops-docs: docs/organizations/security/data-protection.md

Finally, to help combat emerging threats, Azure DevOps employs an "Assume Breach" strategy. A highly specialized group of security experts within Microsoft, known as the Red Team, assumes the role of sophisticated adversaries. This team tests breach detection and response, to accurately measure readiness and the impacts of real-world attacks. This strategy strengthens threat detection, response, and defense of the service. It also allows the team to validate and improve the effectiveness of the entire security program.

> ### Ransomware Attack Protection Overview
> 
> Azure DevOps is built on Azure and leverages Azure's best in class controls to help prevent, detect and/or respond to a ranswomware attack. For more information on how Azure protects customers from Ransomware attacks, please review the [Azure Defenses for Ransomware Attack whitepaper](https://azure.microsoft.com/resources/azure-defenses-for-ransomware-attack/).
> 

Reviewer: ```suggestion
### Ransomware attack protection overview

Azure DevOps is built on Azure and uses Azure's best-in-class controls to help prevent, detect, and respond to a ransomware attack. For more information about how Azure protects customers from ransomware attacks, see [Azure defenses for ransomware attack](https://azure.microsoft.com/resources/azure-defenses-for-ransomware-attack/).

```
Fixing typo and editing for Microsoft style
